### DISCUSSION TITLE
How can cypress be deployed on separate host

### DISCUSSION THREAD

**Discussion by ganijon-dev** [cypress-io/cypress#23990] [Questions and Help]
I have cypress testing set up which is separated from testing project due to we have a lot of forms to test and if we make minor change in project it requires long time to deploy, but we have 1-2 deployments per day. So we decided to separate cypress testing and deploy it on specific url. How can deploy cypress web view to host and run it so developers, business owners and QA can run any tests which was written?
In short how can I run my cypress web view in my domain not in localhost
